packages/gatsby-theme-styleguide/src/gatsby-components/SiteMapData/SiteMapData.js

```diff
@@ -30,6 +30,8 @@ const query = graphql`
           frontmatter {
             title
             order
+            description
+            image
```

Comment:
maybe this method seems better suited : 

https://www.gatsbyjs.org/docs/importing-assets-into-files/#examples

Link to attachments in the frontmatter of your Markdown files:

```
---
title: "Title of article"
attachments:
  - "./assets.zip"
  - "./presentation.pdf"
---
Hi, this is a great article.
```
In an article template component file, you can then query for the attachments:

```
query($slug: String!) {
  markdownRemark(fields: { slug: { eq: $slug } }) {
    html
    frontmatter {
      title
      attachments {
        publicURL
      }
    }
  }
}
```
Read more about querying for files in gatsby-source-filesystem.